scripts/kubestellar-snapshot.sh

```diff
@@ -379,7 +379,7 @@ else
     if [[ -z "$kubeflex_pod" ]]; then
         echoerr "KubeFlex pod not found!"
     else
-        kubeflex_version=$(kubectl --context $helm_context -n kubeflex-system get pod $kubeflex_pod -o json 2> /dev/null | jq -r '.spec.containers[] | select(.image | contains("kubestellar/kubeflex/manager")) | .image' | cut -d':' -f2 || true)
+        kubeflex_version=$(kubectl --context $helm_context -n kubeflex-system get pod $kubeflex_pod -o json 2> /dev/null | jq -r 'try (.spec.containers[] | select(.image | contains("kubestellar/kubeflex/manager")) | .image) catch ""' | cut -d':' -f2 || true)
```

Comment:
Would it be better for the catch expression to give some sort of error message or value? E.g., `catch "unknown"`.